tests/support/globalSetup.ts

```diff
@@ -17,17 +17,17 @@ async function assertHttpApiRunning(): Promise<void> {
   let peerId: string | undefined = undefined;
 
   try {
-    const response = await fetch("http://0.0.0.0:8777");
+    const response = await fetch("http://0.0.0.0:8080/api");
     const data = await response.json();
-    peerId = data.peer.id;
+    peerId = data.node.id;
   } catch (err) {
```

Comment:
we can also use the `8080` port here from `tests/support/fixtures.ts`, thoughts?